### ISSUE TITLE
Right pane is not visible

### ISSUE THREAD

**Issue by mysoultan** [AhmedMah/ClinicManagement#5]
Hello Ahmed, thanks for sharing the clinic system. I was able to deploy the project, but when i am login as an admin. I can't see the management modules such as creating an appointments or access the dashboard.
![image](https://user-images.githubusercontent.com/48092067/104450203-64445a80-55a8-11eb-8b77-66d16758a92d.png)

**Comment by AhmedMah** [comment]
It looks like that the user doesn't belong to any roles, add Administrator and Doctor In AspNetRoles table.

**Comment by Suleman218** [comment]
@AhmedMah Im also facing the same issue can you please explain in detail how to initially add user roles in database for admin etc to get startred

**Comment by pallaviphalke** [comment]
@mysoultan Is your application now ? I need some help on it.

**Comment by AhmedMah** [comment]
I mentioned how to run the project in the **readme** of repository. First open the DB, in **AspNetRoles** table add Administrator and Doctor. Secondly run the project and navigate to http://localhost:xxxx/Account/Register to add admin user. that's it

**Comment by LakshmiKinnera** [comment]
@AhmedMah Hi Ahmed, even after creating roles in AspNetRoles table, im unable to create appointment or access doctors etc., Can you please guide.

**Comment by Suleman218** [comment]
> @AhmedMah Hi Ahmed, even after creating roles in AspNetRoles table, im unable to create appointment or access doctors etc., Can you please guide.

Yes Im also facing the same issue. can you please sort it out @AhmedMah

**Comment by Suleman218** [comment]
> @AhmedMah Hi Ahmed, even after creating roles in AspNetRoles table, im unable to create appointment or access doctors etc., Can you please guide.

@LakshmiKinnera did you reslove this issue?

**Comment by AhmedMah** [comment]
@Suleman218  I tested it, it's working. you have typo in rolenames, the rolenames in AspNetRoles Table  must be "Administrator" and ""Doctor" .
